code/modules/hydroponics/grown/towercap.dm

```diff
@@ -181,17 +181,22 @@
 /obj/structure/bonfire/proc/CheckOxygen()
 	var/turf/T = get_turf(src)
 	var/datum/gas_mixture/G = T.get_readonly_air()
-	if(G.oxygen() > 13)
+	if(G.oxygen() > 7)
```

Comment:
Is there a define for this value anywhere?

Reply:
nothing that I could find. Should I make one in this file?

Reply:
please do, we hate magic numbers